src/nvim/charset.c

```diff
@@ -34,6 +34,16 @@
 
 static int chartab_initialized = FALSE;
 
+static char_u chartab[256];     /* See init_chartab() for explanation */
+
+/* macro version of chartab().
+ * Only works with values 0-255!
+ * Doesn't work for UTF-8 mode with chars >= 0x80. */
+static size_t char_size(size_t c) {
```

Comment:
“Generic” (the one which allows unicode characters) character type is usually `int`, not `size_t`.